shiny/_app.py

```diff
@@ -438,7 +438,13 @@ def _render_page(self, ui: Tag | TagList, lib_prefix: str) -> RenderedHTML:
         ui_res = copy.copy(ui)
         # Make sure requirejs, jQuery, and Shiny come before any other dependencies.
```

Comment:
Move this comment closer to the `.insert()` and add a comment about why we're searching for the Bootstrap dependency